api2-war/src/main/java/eu/europeana/api2/v2/web/controller/ContentReuseFrameworkController.java

```diff
@@ -29,7 +53,12 @@
     @Resource
     private SearchService searchService;
 
-  
+    @Resource(name = "corelib_web_contentReuseFrameworkService")
+    private ContentReuseFrameworkService crfService;
+
+    @Resource(name = "corelib_mediaStorageClient")
```

Comment:
In principle we should only use services here. Seems we are talking to a as "internal" defined client. There should be a general defined service inbetween....